amazon-kinesis-client/src/test/java/software/amazon/kinesis/application/TestConsumer.java

```diff
@@ -67,24 +71,30 @@ public class TestConsumer {
     public TestConsumer(KCLAppConfig consumerConfig) throws Exception {
         this.consumerConfig = consumerConfig;
         this.region = consumerConfig.getRegion();
-        this.streamName = consumerConfig.getStreamName();
-        this.kinesisClient = consumerConfig.buildAsyncKinesisClient();
+        this.streamNames = consumerConfig.getStreamNames();
```

Comment:
looks like we are no longer reading this variable anywhere, now we just call `consumerConfig.getStreamNames()` directly
if we plan on always retrieving the stream names from `consumerConfig`, should we remove this variable altogether?